app/services/add_sandbox_template.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require_dependency "#{Rails.root}/lib/wiki_api"
+require_dependency "#{Rails.root}/lib/wiki_edits"
+
+class AddSandboxTemplate
+  DEFAULT_TEMPLATE = '{{user sandbox}}'
+  def initialize(home_wiki:, sandbox:, sandbox_template:, current_user:)
+    @current_user = current_user
+    @sandbox = sandbox
+    @sandbox_template = sandbox_template
+    @wiki_editor = WikiEdits.new(home_wiki)
+    wiki_api = WikiApi.new(home_wiki)
+    @initial_page_content = wiki_api.get_page_content(@sandbox)
+    add_template
+  end
+
+  private
+
+  def add_template
+    # Never double-post the sandbox template
+    return if sandbox_template_present?
```

Comment:
To really make this an equal branch, it should be
```ruby
if sandbox_template_present?
  return
elsif ...
else ...
end
```

That way, it's super clear that one (and only one) of the three options should happen.

Reply:
Done!